components/ui/Footer.js

```diff
@@ -29,18 +65,31 @@ export const Footer = () => {
       <div className={styles.footerLinks}>
         <h3>Follow us</h3>
         <ul className={styles.footerList}>
+          {SocialMediaList.map((item, index) => (
+            <SocialMediaItem
+              title={item.title}
+              url={item.url}
+              icon={item.icon}
+            ></SocialMediaItem>
+          ))}
+          {/* <li>
+            
+
           <li>
-            <a href="https://facebook.com">Facebook</a>
-          </li>
-          <li>
-            <a href="https://instagram.com">Instagram</a>
-          </li>
-          <li>
-            <a href="https://tiktok.com">Tiktok</a>
+          
+            <Link url={"https://google.com"}>
+              On the streets at night
+            </Link>
           </li>
           <li>
-            <a href="https://google.com">On the streets at night</a>
-          </li>
+            <Link
+              url={
+                "https://www.linkedin.com/in/alireza-hassanloo-415356239/"
+              }
+            >
+              Linkdin
+            </Link>
+          </li> */}
           {/* TASK - React 1 week 2 */}
```

Comment:
It seems like you are missing the pages here.
It should be a list of the available links from the website (just like the navigation bar)